contracts/DebtTokenDeployer.sol

```diff
@@ -33,14 +32,16 @@ contract DeployDebtToken is Ownable{
         uint256 _loanTerm,
         uint256 _loanCycle,
         uint256 _intrestRatePerCycle,
-        address _lender){
+        address _lender)
+    public
+    {
 
         address user = msg.sender;
```

Comment:
There is no need to create a new variable
just use `msg.sender` on lines L#41 and L#44
Additional variable only increases gas cost, nothing else